framework/core/js/src/forum/components/NewAccessTokenModal.tsx

```diff
@@ -0,0 +1,67 @@
+import Modal, { IInternalModalAttrs } from '../../common/components/Modal';
+import Mithril from 'mithril';
+import Button from '../../common/components/Button';
+import app from '../app';
+import AccessToken from '../../common/models/AccessToken';
+import Stream from '../../common/utils/Stream';
+import { SaveAttributes } from '../../common/Model';
```

Comment:
Sort, and use type imports

```suggestion
import app from '../app';
import Modal, { IInternalModalAttrs } from '../../common/components/Modal';
import Button from '../../common/components/Button';
import Stream from '../../common/utils/Stream';
import type AccessToken from '../../common/models/AccessToken';
import type { SaveAttributes } from '../../common/Model';
import type Mithril from 'mithril';
```